compoments/report-repair/contact-us.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
```

Comment:
I'm wondering if it would be worth putting all the files with the components for when the standard repair is not happening (for example `contact us`, `emergency repair`, `not eligible communal repairs`) in a separate folder under `report-repair` for better visibility. What do you think?

Reply:
I think that categorising the components would be useful.

Your suggestion would be to have "early exit" pages within a subfolder is a good start.
Other categories could be "tenant", "leaseholder" and "communal" specific components. Then common components could be at either the root or in a "common" folder.

Beyond the scope of this ticket though, please create a ticket for this work.